From 764ae876c621489d99a2e8f50ef8660d6eb173d0 Mon Sep 17 00:00:00 2001
From: Danny Lin <danny@kdrag0n.dev>
Date: Fri, 18 Sep 2020 23:02:09 -0700
Subject: [PATCH 11/15] colors: Use iOS blue accent color

This is similar to the stock Pixel blue accent color, but it's slightly
brighter and less saturated, which looks better in my opinion.

Change-Id: I6040e7500693aa0a5904026bfcec2bcdc185c87f
---
 core/res/res/values/colors_device_defaults.xml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/core/res/res/values/colors_device_defaults.xml b/core/res/res/values/colors_device_defaults.xml
index 7a8f411992c..db44e4b28e3 100644
--- a/core/res/res/values/colors_device_defaults.xml
+++ b/core/res/res/values/colors_device_defaults.xml
@@ -33,8 +33,8 @@
     <color name="tertiary_device_default_settings">@color/tertiary_material_settings</color>
     <color name="quaternary_device_default_settings">@color/quaternary_material_settings</color>
 
-    <color name="accent_device_default_light">@color/accent_material_light</color>
-    <color name="accent_device_default_dark">@color/accent_material_dark</color>
+    <color name="accent_device_default_light">#007CFA</color>
+    <color name="accent_device_default_dark">#0086FA</color>
     <color name="accent_device_default">@color/accent_device_default_light</color>
 
     <color name="background_device_default_dark">@color/background_material_dark</color>
-- 
2.28.0

